src/compilers/project.json

```diff
@@ -0,0 +1,25 @@
+{
+    "version": "1.0.0-*",
+    "compilationOptions": {
+        "emitEntryPoint": true
+    },
+
+    "dependencies": {
+      "Microsoft.NETCore.App": {
+        "type":"platform",
+        "version":"1.0.0-rc2-23925"
+      },
+      "Microsoft.CodeAnalysis.CSharp": "1.3.0-beta1-20160321-04",
+      "Microsoft.Net.Compilers.netcore": "1.3.0-beta1-20160321-04",
+      "Microsoft.Net.CSharp.Interactive.netcore": "1.3.0-beta1-20160321-04",
+      "Microsoft.FSharp.Compiler.netcore": "1.0.0-alpha-160318",
+      "Microsoft.DiaSymReader.Native": "1.3.3",
+    },
+    "frameworks": {
+      "netstandard1.5": {
```

Comment:
Shouldn't this be `netcoreapp1.0`? @anurse now that we support `type:platform` what is there anything preventing us from using `netcoreapp1.0` now?